node/node.go

```diff
@@ -177,6 +180,26 @@ func (n *Node) Run(ctx context.Context) {
 
 	n.services = []service.Service{synchronizer, http}
 
+	if n.cfg.EthNode == "" {
+		n.log.Warnw("Ethereum node address not found; will not verify against L1")
+	} else {
+		var coreContractAddress common.Address
+		coreContractAddress, err = n.cfg.Network.CoreContractAddress()
+		if err != nil {
+			n.log.Errorw("Error finding core contract address for network", "err", err, "network", n.cfg.Network)
+			return
+		}
+		var ethSubscriber *l1.EthSubscriber
+		ethSubscriber, err = l1.NewEthSubscriber(n.cfg.EthNode, coreContractAddress)
+		if err != nil {
+			n.log.Errorw("Error creating ethWatcher", "err", err)
+			return
+		}
+		confirmationPeriod := 16
```

Comment:
`confirmationPeriod` is for number of blocks right? Also since it's hardcoded now maybe make it const (we can either keep it here or move it outside the func